sqlmodel/main.py

```diff
@@ -462,55 +491,21 @@ def get_column_from_field(field: ModelField) -> Column:  # type: ignore
 default_registry = registry()
 
 
-def _value_items_is_true(v: Any) -> bool:
-    # Re-implement Pydantic's ValueItems.is_true() as it hasn't been released as of
-    # the current latest, Pydantic 1.8.2
-    return v is True or v is ...
-
-
-_TSQLModel = TypeVar("_TSQLModel", bound="SQLModel")
-
-
 class SQLModel(BaseModel, metaclass=SQLModelMetaclass, registry=default_registry):
```

Comment:
In the test `tests\test_tutorial\test_delete`, the following error occurred: `AttributeError: 'Hero' object has no attribute '__pydantic_extra__'`.

You can solve this problem by adding the following code to SQLmodel:
```python
def __new__(cls, *args: Any, **kwargs: Any) -> Any:
    new_object = super().__new__(cls)
    object.__setattr__(new_object, "__pydantic_fields_set__", set())
    return new_object
```